shard.yml

```diff
@@ -7,3 +7,8 @@ authors:
 crystal: ">= 0.35.1, < 2.0.0"
```

Comment:
Since we're removing 0.35.1 from the testing matrix, we should also but this line to 0.36.1